src/helpers/responseMessage.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Generic to response message
+ * @function
+ *
+ * @param {string} response
+ * @param {string} statusCode
+ * @param {string} object
+ *
+ * @returns {Void} undefined
+ * @description global response message to users
+ */
+const responseMessage = (response, statusCode, object) => response.status(statusCode).json({
```

Comment:
Good job here. However, I'm a bit confused as to why the function name is `responseMessage` and the status is `bad request`. Please look into this as I'm unable to understand why this needed to be abstracted out.